tests/cases/fourslash/importNameCodeFixConvertTypeOnly1.ts

```diff
@@ -9,5 +9,5 @@
 //// new B
 
 goTo.file('/b.ts');
-verify.importFixAtPosition([`import { B, type A } from './a';
+verify.importFixAtPosition([`import { type A, B } from './a';
```

Comment:
This file doesn't have any preferences set, but we're now switching from auto inserting the type import at the end to inlining it; is that desirable / by design?

I of course prefer inline sorting as that's what other tools do, but I'm just thinking about if this will be a surprise for someone downstream like vscode.

Reply:
I'm just half confused because `tests/cases/fourslash/importNameCodeFix_importType7.ts` appears to not detect any order at all but this example doesn't seem much different.

Reply:
`importFixes` is incorrectly comparing named imports when doing type promotions, I think that other case may be correct only by chance.

Reply:
these changes have been reverted and importfixes has been corrected!